multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -1772,21 +1778,23 @@ def _default_predict(
             blacklist_msgs.append("HPU available")
             blacklist_msgs.append("select gpus")
             blacklist_msgs.append("LOCAL_RANK")
+            blacklist_msgs.append("Trainer(barebones=True)")
         log_filter = LogFilter(blacklist_msgs)
 
         with apply_log_filter(log_filter):
             evaluator = pl.Trainer(
                 accelerator="gpu" if num_gpus > 0 else "auto",
-                devices=get_available_devices(num_gpus=num_gpus, auto_select_gpus=self._config.env.auto_select_gpus),
+                devices=num_gpus if num_gpus > 0 else "auto",
                 num_nodes=self._config.env.num_nodes,
                 precision=precision,
                 strategy=strategy,
                 benchmark=False,
-                enable_progress_bar=self._enable_progress_bar,
+                enable_progress_bar=False if barebones else self._enable_progress_bar,
```

Comment:
I think this is disabled on lightning side so we don't have to apply the logic explicitly. https://lightning.ai/docs/pytorch/stable/common/trainer.html

Reply:
I found that It would throw an error if both enable_progress_bar and barebones are set True.